site2/website/versioned_docs/version-2.8.0/functions-overview.md

```diff
@@ -35,7 +35,7 @@ Pulsar Functions provide a wide range of functionality, and the core programming
 
   * Apply some processing logic to the input and write output to:
     * An **output topic** in Pulsar
-    * [Apache BookKeeper](#state-storage)
+    * [Apache BookKeeper](functions-develop.md#state-storage)
```

Comment:
@Anonymitaet Note this modification since the pr about 2.8.0 versions doc was merged after my last pr.